DEPS

```diff
@@ -8,6 +8,7 @@ deps = {
   "vendor/python-patch": "https://github.com/brave/python-patch@d8880110be6554686bc08261766538c2926d4e82",
   "vendor/omaha":  "https://github.com/brave/omaha.git@aa5d8f956d6b4fc929d835995b25df159af6dcf9",
   "vendor/sparkle": "https://github.com/brave/Sparkle.git@07933da3e178265d0f0ba86e02bbde38e701a04d",
+  "vendor/guardian-connect": "https://github.com/simonhong/GuardianConnect.git@1b042e4e248c0c712970c123076d51d49b50fc24",
```

Comment:
Can we use https://github.com/brave/GuardianConnect instead?

Reply:
Also, please put in `third_party/` instead of `vendor/` since the latter is deprecated.

Reply:
@fmarier I didn't know about `third_party`. Thanks for noticing this.
Yup, I'll use https://github.com/brave/GuardianConnect after merging some changes to it.